tfe/resource_tfe_team_members_test.go

```diff
@@ -2,23 +2,30 @@ package tfe
 
 import (
 	"fmt"
+	"os"
 	"testing"
 
 	tfe "github.com/hashicorp/go-tfe"
 	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccTFETeamMembers_basic(t *testing.T) {
 	users := []*tfe.User{}
+	username := os.Getenv("TFE_MEMBER_USERNAME")
```

Comment:
I think I prefer something like `TFE_USER1` and `TFE_USER2` to keep it a bit shorter and to allow them to be used in other places as well.

Reply:
Sure, not a problem at all - I will rename these variables. 👍